qml/components/AppInformation.qml

```diff
@@ -0,0 +1,125 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+
+BackgroundItem {
+    property bool _expanded: false
+
+    property var pkg
+
+    id: infoItem
+    width: parent.width
+    height: content.height
+    clip: true
+    highlighted: down && !highlightDelayTimer.running
+    _showPress: highlighted
+    propagateComposedEvents: true
+
+    onPressedChanged: {
+        if (pressed) {
+            highlightDelayTimer.restart()
+        }
+    }
+
+    onClicked: _expanded = !_expanded
+
+    Behavior on height { NumberAnimation { duration: 200; easing.type: Easing.InOutQuad } }
+
+    Timer {
+        // Idea is from the official store client
+        id: highlightDelayTimer
+        interval: 50
+    }
+
+    Column {
+        id: content
+        anchors {
+            left: parent.left
+            right: parent.right
+            leftMargin: Theme.horizontalPageMargin
+            rightMargin: Theme.horizontalPageMargin
+        }
+
+        Item {
+            width: parent.width
+            height: bodyLabel.height
+
+            Label {
+                id: bodyLabel
+                width: parent.width
+                maximumLineCount: _expanded ? 0 : 7
+                text: pkg.description
+                color: infoItem.highlighted ? Theme.highlightColor : Theme.primaryColor
+                linkColor: Theme.highlightColor
+                font.pixelSize: Theme.fontSizeSmall
+                wrapMode: Text.WordWrap
+                onLinkActivated: openLink(link)
+            }
+
+            OpacityRampEffect {
+                sourceItem: bodyLabel
+                enabled: !_expanded
+                direction: OpacityRamp.TopToBottom
+            }
+        }
+
+        Item {
+            id: spacer
+            visible: _expanded
+            height: Theme.paddingMedium
+        }
+
+        ChumDetailItem {
```

Comment:
I would move that on top of description, out of AppInformation. Other elements are good to stay in here, it looks nice.